### PR TITLE
Better intergration tests

### LINKED ISSUE #45: Adding behat integration tests for the OCS API
- [x] @nickvergessen Write OCS endpoint for "if debug === true" to add notifications for app "testing*"
- [x] @nickvergessen Write notification notifier for "if debug === true" to parse notifications for app "testing*"
- [ ] @SergioBertolinSG Write tests to read and delete notifications

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Depends on https://github.com/owncloud/notifications/pull/53
- [x] Add tests for empty notification list
- [x] Add test to get all notifications
- [x] Add test to get one notification
- [x] Add test to delete one notification

Fix #45

### CODE REVIEW

File: appinfo/routes.php [hunk 1]
```diff
@@ -39,3 +39,38 @@
...
+if (\OC::$server->getConfig()->getAppValue('notifications', 'debug')) {
+	$controller = new \OCA\Notifications\Tests\Integration\Controller(
+		'notifications',
+		\OC::$server->getRequest(),
+		\OC::$server->getConfig(),
+		\OC::$server->getNotificationManager()
+	);
+	\OCP\API::register(
+		'post',
+		'/apps/notifications/testing/notifiers',
```
Comment (nickvergessen) [inline appinfo/routes.php]: should use capabilities instead
